nixos/modules/hardware/openrgb.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.openrgb;
+in
+
+{
+  options.programs.openrgb = {
+    enable = mkEnableOption ''
+      OpenRGB support. Will install udev rules to allow for more devices
+      to be shown, and optionally run OpenRGB as a server.
```

Comment:
FYI `mkEnableOption` includes a trailing period so you can omit yours.

Reply:
Thanks